src/main/webapp/components/utils/getField.js

```diff
@@ -7,7 +7,9 @@
  * @param defaultValue the value to be used if obj[field] === undefined.
  */
 export const getField = (obj, field, defaultValue) => {
-  return obj[field] !== undefined ?
-      obj[field] :
-      (defaultValue !== undefined ? defaultValue : null);
+  return obj !== undefined
```

Comment:
the idiomatic pattern is using optional chaining:

```
return obj?.[field] ?? defaultValue;
```

Note that this requires a fairly recent browser.

Reply:
... alternatively, I'd avoid ternaries for better readability and just add more `if` statements:

```
if (!obj) return defaultValue;
if (obj[field] == null) return defaultValue;
...
```

The precedence and association of the cases is much more readable than a ternary in this mode.

Reply:
Thanks for the insight, learnt something about combining optional chaining with the nullish coalescing operator 😃. 

Based on your other comment, this method might be deprecated in the future. If not, I'll most likely default to a series of <code>if</code> statements since it is far more readable.